sdk/android/ooyalaSkinSDK/src/main/java/com/ooyala/android/ooyalaskinsdk/BridgeMessageBuilder.java

```diff
@@ -18,8 +18,8 @@
  * Created by michael.len on 2/1/16.
  */
 class BridgeMessageBuilder {
-
-  public static WritableMap buildTimeChangedEvent (OoyalaPlayer player){
+  public static String nextVideoEmbedCode;
```

Comment:
why do you need to store this here? no storing anything in the BridgeMessageBuilder - it is entirely a static method class. no state in it